jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -11,34 +11,176 @@ contentTags:
 :toc: macro
 :toc-title:
 
-The CircleCI VS Code extension helps you avoid context-switching between VS Code and your browser.
+The CircleCI Visual Studio Code extension enables users to manage their CircleCI pipelines directly from VS Code. 
 
 [#introduction]
 == Introduction
 
-The CircleCI VS Code extension lets you visualize and manage CircleCI pipelines directly from your IDE, as well as providing contextual help when creating, modifying and editing CircleCI YAML config files. 
+The VS Code extension includes:
 
-The extension allows you to:
+- The **Pipeline Manager**, which displays pipeline status live within the integrated development environment (IDE). The pipeline manager allows you to identify issues and take immediate action on your pipelines without context-switching between VS Code and your browser.
+- The **Config Helper**, which provides in-file help with navigating, writing, and editing configuration files.
 
-* Authenticate and connect VS Code to CircleCI.
-* Browse and interact with your pipelines. 
+[#install-the-vs-code-extension]
+== Install the VS Code extension
 
-Available actions include:
+For installation and setup instructions, visit the VS Code link:https://marketplace.visualstudio.com/items?itemName=circleci.circleci[Marketplace page], or search for the official CircleCI extension on the Extensions tab in the VS Code app. 
 
-* View pipelines, workflows, jobs statuses
-* View test outputs
-* Download artifacts
-* Approve jobs
-* Re-run builds
-* Be alerted through notifications when your workflows change status or need your attention
-* Access in-file support when editing CircleCI YAML configuration files, including:
-** Syntax validation
-** Syntax highlighting
-** Go-to-definition and go-to-reference
-** On-hover documentation and usage hints
-** Autocompletion
+[#pipelines-panel]
+== Pipelines panel
+The pipelines panel provides a visual interface for managing your pipelines. The panel lists your most recent pipelines, workflows, and jobs in a project, and lets you monitor their status as well as interact with them.
 
-[#install-the-vs-code-extension]
-== Install the VS Code extension
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_pipelines-panel.png[Screenshot of side panel with pipeline information]
+
+For each object, you can trigger certain events on hover.
```

Comment:
Could we move this above the screenshot? Just looks a bit lost under to me - just a suggestion.

Reply:
Also, I see you've left notes to check this on hover functionality - I don't see it for pipelines and workflows, apart from to rerun a workflow, I see a symbol to open up each job in a new window and can open the job in browser from there using a button. Maybe I'm using an outdated version 🤔

Reply:
Yes - @BeFunes this is what I was referring to with the "check on hover" comment. For pipelines, I don't see anything on hover, and for workflows, I don't see the option to open in browser, just the option to rerun:
https://user-images.githubusercontent.com/36839689/230639355-de6a8586-55bb-4c81-8842-a9d38f8e37d7.mov

Reply:
😨 You are right, someone removed it!! 
I've just added a ticket to re-introduce them for pipelines and workflows, so I think you can leave it there for those maybe? 
Not necessary for the job. 

Also, we need to add "Rerun workflow from fail" for workflows, which is new

Reply:
As for the image, In my opinion it's helpful, however it's positioned very weirdly. 
It should come BEFORE the sentence "For each object..." 
(like [here](https://marketplace.visualstudio.com/items?itemName=circleci.circleci))

Reply:
@BeFunes ah, I actually moved the image based on @rosieyohannan comment above. I'm fine with either. Let me know!

Reply:
Feel free to move it back!